src/backend/distributed/commands/collation.c

```diff
@@ -588,9 +590,12 @@ PostprocessDefineCollationStmt(Node *node, const char *queryString)
 	ObjectAddress collationAddress =
 		DefineCollationStmtObjectAddress(node, false);
 
-	EnsureDependenciesExistOnAllNodes(&collationAddress);
+	if (IsObjectDistributed(&collationAddress))
```

Comment:
Not directly related to here, but it seems we can do the following from the workers on the distributed objects, are they expected?
```SQL
-- public is a distributed object
alter schema public owner to onderkalaci ;
```

Can you please go over the objects & ops to see if there are any such holes where we need to add `EnsureCoordinator`?

Reply:
We support this on the worker as customers need to handle access control for roles in community by themselves, so it is expected.